Dockerfile

```diff
@@ -13,6 +13,6 @@ ARG GIT_BRANCH
 
 RUN pub get
```

Comment:
you should be able to remove this, because `dart_build_image` already doing it under the hood